babel.config.js

```diff
@@ -52,9 +76,6 @@ module.exports = {
     production: {
       plugins: [
         ...plugins,
-        // console.log is expensive for performance on native
-        // we don't want it on web either, but it's useful for development
-        ['transform-remove-console', { exclude: ['error', 'warn'] }],
```

Comment:
Why we no longer want that?

Reply:
I don't think it actually did anything, AFAIR all console.logs showed up anyway. And anyway, all logging goes through `logger`, and not directly to console.log, so applications can choose whether or not to silence logs